ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -63,6 +63,7 @@ Item {
 
         readonly property string stateLoginRegularUser: "regularUserLogin"
         readonly property string stateLoginKeycardUser: "keycardUserLogin"
+        property bool isRegularLogin: image.source.toString().includes("status-logo")
```

Comment:
`readonly` and ewwww ..., there should be a better way to distinguish between the login types, @saledjenic any idea? :)

Reply:
@saledjenic is (root.startupStore.currentStartupState.stateType === Constants.startupState.login) always returning to this 
<img width="167" alt="l" src="https://github.com/status-im/status-desktop/assets/31625338/32d1e18f-51e6-4de0-ae75-7cbebff35507">

image?

Reply:
> readonly and ewwww ..., there should be a better way to distinguish between the login types, @saledjenic any idea? 

Of course there is, try with `root.startupStore.selectedLoginAccount.keycardCreatedAccount` it is true if selected account to login is a keycard account, otherwise it's false.